net-analyzer/prelude-correlator/prelude-correlator-4.0.0.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit systemd distutils-r1
+
+DESCRIPTION="Real time correlator of events received by Prelude Manager"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/setuptools"
```

Comment:
Missing PYTHON_USEDEP.

Reply:
PYTHON_USEDEP shouldn't be defined by packager, what do you want ?

Reply:
Are you sure PYTHON_USEDEP have to be defined in the ebuild ?

Reply:
Not defined, used in setuptools dep.

Reply:
indeed, sorry